### PR TITLE
Autogen bookmarks from README.md w/ script

### PR DESCRIPTION
**Pull request by ScriptAutomate**
- New bookmark file added to repo
- New bookmark generator Python file that only requires Python, no other dependencies
- Updated some SaltStack -> Salt Project branding

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 # Awesome SaltStack [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-> A collaborative curated list of awesome SaltStack resources, tutorials and other salted stuff.
+> A collaborative curated list of awesome Salt Project resources, tutorials and other salted stuff.
+
+> **NOTE: A bookmark file is available with all of these links, ready for import into your web browser. Download the `awesome-saltstack-bookmarks.html` file from the root of this repo!**
 
 <!--lint ignore double-link-->
-[<img src="https://user-images.githubusercontent.com/519955/35341388-d8c0cf0e-0125-11e8-9831-51f13fab58c7.jpg" width="100%">](https://www.saltstack.com/)
+[<img src="https://gitlab.com/saltstack/open/salt-branding-guide/-/raw/master/logos/SaltProject_altlogo_teal.png" width="100%">](https://saltproject.io/)
```
Comment (hbokh) [inline README.md]: Ah... new logo.

File: bkmark-generator.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+##
```
Comment (hbokh) [inline bkmark-generator.py:1]: I'd like to have the `*.html` and `*.py` files separated from the current rootdir into a new folder named "bookmarks". Would that be possible?

**Comment by hbokh** [comment]
@ScriptAutomate / Derek, judging from your profile: I wouldn't mind to have VMware / Salt take over this repository.

**Comment by ScriptAutomate** [comment]
> @ScriptAutomate / Derek, judging from your profile: I wouldn't mind to have VMware / Salt take over this repository.

@hbokh I'll bring this up with the Salt Project team, and get back to you.

**Comment by ScriptAutomate** [comment]
> @ScriptAutomate / Derek, judging from your profile: I wouldn't mind to have VMware / Salt take over this repository.

@hbokh  Can you invite my user account on as a collaborator under the repo settings, in the meantime, as I continue to seek out the process for taking ownership of this repo if Salt Project is able to?

**Pull request closed without merging**